common/src/main/java/fudge/notenoughcrashes/stacktrace/CrashUtils.java

```diff
@@ -13,7 +13,7 @@
 
 public final class CrashUtils {
 
-    private static final Logger LOGGER = LogManager.getLogger("TMC");
+    private static final Logger LOGGER = LogManager.getLogger("NEC");
```

Comment:
Delete this logger and use `NotEnoughCrashes.LOGGER` instead

Reply:
Fixed!